s3same/__init__.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import travispy
+import yaml
+
+from .travis import travis_encrypt as _travis_encrypt
```

Comment:
If I'm understanding you correctly, you can specify things that are meant for export in your `__init__.py` using the [`__all__` variable](https://docs.python.org/2/tutorial/modules.html#importing-from-a-package). Technically this just dictates what gets imported if someone did, say, `from s3same import *`, but it's also a signal to other developers (like the `_` prefix) that anything else is private and subject to change.